### PR TITLE
[Impeller] Update buildroot with json.hpp and tinygltf

### PR DESCRIPTION
**Pull request by bdero**
Buildroot PR: https://github.com/flutter/buildroot/pull/627

### CODE REVIEW

File: ci/licenses_golden/licenses_third_party [hunk 1]
```diff
@@ -27249,6 +27251,95 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
...
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+====================================================================================================
+
+====================================================================================================
+LIBRARY: tinygltf
+ORIGIN: ../../../third_party/tinygltf/json.hpp
```
Comment (chinmaygarde) [inline ci/licenses_golden/licenses_third_party:27290]: We already have a copy of this JSON dependency. Lets use that instead and ignore this one.

File: tools/licenses/lib/main.dart [hunk 2]
```diff
@@ -1021,6 +1021,8 @@ class _RepositoryDirectory extends _RepositoryEntry implements LicenseSource {
     return !entry.fullName.endsWith('third_party/gn') &&
            !entry.fullName.endsWith('third_party/gradle') &&
            !entry.fullName.endsWith('third_party/imgui') &&
+           !entry.fullName.endsWith('third_party/tinygltf/examples') &&
```
Comment (chinmaygarde) [inline tools/licenses/lib/main.dart:1024]: GLTF isn't going to be used at runtime in any vended tool right now. Let's ignore the whole thing.

Comment (bdero) [reply tools/licenses/lib/main.dart]: Done.

File: ci/licenses_golden/licenses_third_party (hunk 1 above)
Comment (bdero) [reply ci/licenses_golden/licenses_third_party]: Looks like it's not a top level dependency -- the compiler is getting it from `//third_party/inja/third_party/include`. Untangling this at the moment.

Comment (bdero) [reply ci/licenses_golden/licenses_third_party]: Added `nlohmann/json` config to the buildroot PR and got rid of the 3rd party includes in inja. Should be good to go once I fix up the licenses.

**Review by chinmaygarde — APPROVED** [review]

**Pull request merged**